api/v1alpha1/condition_consts.go

```diff
@@ -57,6 +57,17 @@ const (
 	GuestCustomizationFailedReason = "GuestCustomizationFailed"
 )
 
+const (
+	// VirtualMachineToolsCondition exposes the status of VirtualMachine tools running in the guest OS, when available
```

Comment:
Isn't "VMware Tools" more commonly used, as in the product name and what the VC UI uses? I don't think I've ever seen this spelled out as "VirtualMachine tools"

Reply:
yeah, VMware tools is more commonly used as the product name. I would like to use that in the comments too. Any restrictions from using "VMware" in our api docs?.

Reply:
Should be fine to use that, after all, this is a product of VMware.